snap/snapcraft.yaml

```diff
@@ -12,10 +12,17 @@ description:        |
 grade:              stable
 confinement:        strict
 type:               app
+base:               core18
 
 apps:
   krop:
     command:        desktop-launch krop
+    extensions: 
+      - kde-neon
+    command-chain:
+      - snap/command-chain/snapcraft-runner
+    environment:
+      HOME: $SNAP_USER_COMMON
     plugs:
       - desktop
```

Comment:
`desktop`, `desktop-legacy` and `x11` are not needed anymore, these are added by the `kde-neon` extension.